### ISSUE TITLE
Build on windows : failed to run a maven command

### ISSUE THREAD

**Issue by NPavie** [daisy/pipeline-assembly#211]
When trying to run the makefile with the make executable on windows, the compilation fails on this call: 
```
exec(env("JAVA_HOME", "C:/Users/npavie/Nextcloud/Devs/pipelineui/pipeline-ui/engine/src/main/jre/OpenJDK11U-jdk_x64_windows_hotspot_11.0.13_8/jdk-11.0.13+8"),               \
     "mvn", "package", "-Pbuild-jre-win64");
```
The exec cannot find the mvn command.

On windows, the exact executable name of maven is "mvn.cmd".
Build works if i change the mvn command by mvn.cmd in the makefile.

**Comment by bertfrees** [comment]
Hmm I thought I had already fixed this.

**Comment by bertfrees** [comment]
@NPavie Which version are you testing? It should have been fixed in 84a43ab7b1f2cd31cce5689720f3ab2ad5b89c67.

**Comment by NPavie** [comment]
Oups, i was not on master, i was on the new-ui branch, my bad !
